src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        Color color = Color.values()[new Random().nextInt(Color.values().length)];
```

Comment:
Creating a new instance of `Random` every time the method is called is not efficient. Consider creating a single `Random` instance as a class field and reusing it.